requirements/bench.txt

```diff
@@ -2,6 +2,8 @@ requests>=2.21.0
 xlwt>=1.3.0
 xlrd>=1.2.0
 matplotlib>=2.2.3
+python-dev-tools 
```

Comment:
that's an aweful lot of stuff for just one use case, is that really needed?

Reply:
removed